### ISSUE TITLE
Implement learning rate scheduling

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#87]
Implement **ReduceLROnPlateau** learning rate scheduling in the training script.
